features/customer.feature

```diff
@@ -70,24 +70,26 @@ Scenario: Search for john
     And I should not see "sally" in the results
     And I should not see "daisy" in the results
 
-# Scenario: Update a Pet
-#     When I visit the "Home Page"
-#     And I set the "Name" to "fido"
-#     And I press the "Search" button
-#     Then I should see the message "Success"
-#     And I should see "fido" in the "Name" field
-#     And I should see "dog" in the "Category" field
-#     When I change "Name" to "Loki"
-#     And I press the "Update" button
-#     Then I should see the message "Success"
-#     When I copy the "Id" field
-#     And I press the "Clear" button
-#     And I paste the "Id" field
-#     And I press the "Retrieve" button
-#     Then I should see the message "Success"
-#     And I should see "Loki" in the "Name" field
-#     When I press the "Clear" button
-#     And I press the "Search" button
-#     Then I should see the message "Success"
-#     And I should see "Loki" in the results
-#     And I should not see "fido" in the results
+# Scenario: Update a Customer
```

Comment:
This test looks great, but you will also have to implement the actual update feature. If you take a look at `rest_api.js` ln88-101 you'll see that the fields are still messed up from the pet use case. once you update them to reflect the customer-specific fields, we can run this test again

Reply:
Thanks Noah. I'll fix that and resend.

Reply:
I have committed the new updates. Please review for merge